devicetypes/smartthings/zwave-mold-detector.src/zwave-mold-detector.groovy

```diff
@@ -0,0 +1,205 @@
+/**
+ *  Copyright 2020 SmartThings
```

Comment:
2021 here and below, right?

Reply:
Well, Zuza did most of the work back in 2020.